spotbugs/src/main/java/edu/umd/cs/findbugs/util/ClassName.java

```diff
@@ -265,13 +267,26 @@ public static boolean matchedPrefixes(String[] classSearchStrings, @DottedClassN
         }
 
         for (String p : pp) {
-            if (p.length() > 0 && className.indexOf(p) >= 0) {
+            if (p.length() > 0 && (StringUtils.containsIgnoreCase(className, p) || fuzzyMatch(className, p))) {
                 return true;
             }
         }
 
         return false;
+    }
 
+    /**
+     * Perform a fuzzy matching, by comparing the Levenshtein distance of the
+     * simple class name and the search string.
+     * 
+     * @param className the full class name
+     * @param searchString the search string
+     * @return true, if the strings are similar, false otherwise
+     */
+    private static boolean fuzzyMatch(String className, String searchString) {
+        // compare to a maximum Levenshtein distance of 3
```

Comment:
What is Levenshtein distance 3? How does it compare to other distances? Please describe or link to a description.